contracts/ContractRegistry.sol

```diff
@@ -2,15 +2,27 @@ pragma solidity 0.5.16;
 import "./spec_interfaces/IContractRegistry.sol";
 import "./IContractRegistryListener.sol";
 import "./WithClaimableRegistryManagement.sol";
+import "./spec_interfaces/ILockable.sol";
+import "./Initializable.sol";
 
-contract ContractRegistry is IContractRegistry, WithClaimableRegistryManagement {
+contract ContractRegistry is IContractRegistry, Initializable, WithClaimableRegistryManagement {
 
 	mapping (string => address) contracts;
 	address[] managedContractAddresses;
 
 	mapping (string => address) managers;
 
-	function setContract(string calldata contractName, address addr, bool managedContract) external onlyRegistryManager {
+	modifier onlyManager {
+		require(msg.sender == registryManager() || msg.sender == initializationManager(), "sender is not an admin (registryManager or initializationManager)");
```

Comment:
the use of initialization completed -> initializationManager = address(0) is not reflected from the modifier and is a bit difficult to understand. 
I'm trying to think about a simple solution, at minimum, can change the text:
sender is not registryManager or initializationManager and initialization in progress.

Reply:
Will change.
Initialization is complete by renouncing the ownership, isn't this a very common pattern?

Reply:
It is a common pattern, but it's not clear that the Initialisation manager renounces itself, i think a change in the text along with the getter is ok